pennylane/beta/tapes/tape.py

```diff
@@ -106,7 +413,7 @@ def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
                 # as a stopping condition
                 stop = stop or isinstance(obj, qml.beta.queuing.MeasurementProcess)
 
-            if stop:
+            if stop or isinstance(obj, BatchTape):
```

Comment:
For now stop at `BatchTapes` for simplicity, but of course we would eventually move expand everything in the `BatchTape` too...